client/app/utils/jsonSchema/operations.ts

```diff
@@ -44,28 +41,46 @@ const operationPage1: RJSFSchema = {
       },
       title: "Regulated Product Name(s)",
     },
-    reporting_activities: {
-      type: "array",
-      items: {
-        type: "number",
-      },
-      title: "Reporting Activities",
-    },
-    "Did you submit a GHG emissions report for reporting year 2022?": {
-      type: "boolean",
-      default: false,
-    },
-    multiple_operators_section: {
+    // Commenting this out as it is not needed for MVP
+    // reporting_activities: {
+    //   type: "array",
+    //   items: {
+    //     type: "number",
+    //   },
+    //   title: "Reporting Activities",
+    // },
+    // process_flow_diagram: {
+    //   type: "string",
+    //   title: "Process Flow Diagram",
+    //   format: "data-url",
+    // },
+    // boundary_map: {
+    //   type: "string",
+    //   title: "Boundary Map",
+    //   format: "data-url",
+    // },
```

Comment:
I'd recommend deleting the `process_flow_diagram` and `boundary_map` parts entirely - I don't think we'll ever be using them